cfme/tests/cloud/test_provisioning.py

```diff
@@ -47,14 +46,12 @@ def testing_instance(request, setup_provider, provider, provisioning, vm_name):
         'first_name': 'Image',
         'last_name': 'Provisioner',
         'notes': note,
-        'instance_type': provisioning['instance_type'] if provider.type != "azure" else
-        provisioning['vm_size'],
```

Comment:
are you sure such options don't need to be present for OpenStackProvider, GCEProvider ?